data/remnant missions.txt

```diff
@@ -1252,3 +1252,240 @@ mission "Remnant: Expanded Horizons Quarg 3"
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Excellent!" he glances at his datapad, "there are several teams working on aspects of the jump drive, and the one with the highest priority for this one is on <planet>. I will notify them to expect you."`
+				accept
+			label refuse
+			`	He considers this, and sings, "If you wish. Our offer stands, whenever you choose to return."`
+				defer
+	on complete
+		conversation
+			`When you arrive on <planet> there is a team of engineers and mechanics waiting for you, along with a flatbed freight truck. Once you open the cargo doors they quickly free the broken jump drive from its moorings and load it delicately onto their truck. The lead engineer comes over to hand you <payment>. "Thank you for salvaging this. I suspect my compatriot who spoke to you never mentioned this, but we have a fund set aside for rewarding captains who bring in rare pieces of alien technology. If you find any more of these drives, there is sure to be a research team somewhere that is willing to reward you for it."`
+			choice
+				`	"I will keep a lookout for more."`
+					goto end
+				`	"Aren't you going to replace it with a hyperdrive?"`
+			`	The engineer looks at you incredulously. "You mean you were actually depending on this drive?" He shakes his head in disbelief. "You are lucky you managed to get this far, then. It is surprising that they exist at all, let alone still work properly. If you need a replacement drive, there are plenty available in the outfitter."`
+			label end
+			`	"Next time you find one of these drives, just check with a starport director. They will know where it will need to go. A word of warning though; I strongly recommend not using these drives - there may disappear at the most inconvenient of times, and there won't always be an outfitter on hand." He turns and heads off with the rest of the team, turning briefly to give you friendly wave as they disembark.`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 2"
+	name "Broken Jump Drive Delivery"
+	description "The researchers on <planet> have requested your broken jump drive."
+	to offer
+		has "Remnant: Broken Jump Drive 1: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		attributes outfitter
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`Remembering the advice given by the engineer who paid you for the broken jump drive, you stop by the starport director's desk. Information must flow quickly amongst the Remnant, because by the time you arrive at the director's desk she already has the information ready for you.`
+			`	"Greetings, Newcomer," she chants. "Are you interested in turning in the broken jump drive you salvaged to our researchers?"`
```

Comment:
Should Newcomer be capitalized here? It's not really a title.

Reply:
I capitalized it to convey emphasis. She is using it like a title, and putting emphasis on the word in the manner that capitalizing it implies. She isn't just saying it casually as a convenient shorthand for referring to `<first> <last>`

Reply:
If newcomer is to be capitalized here, then you'll need to look at the other places it has been used. I can understand newcomer being used like captain, so if you're the only newcomer that can be referred to, all uses of newcomer should be capitalized.

Reply:
OK. Does the following make sense: When it is used as a general descriptor, it isn't capitalized. If it is used as a pseudo title, it is capitalized. Does that work?

Example of the former: "we saw the news that the newcomer among us was..."

Example of the later: "Greetings, Newcomer!"

Reply:
This is the only place where newcomer is capitalized as a title right now. I'd rather just make this one lower cased and keep it that way.